src/sentry/shared_integrations/exceptions.py

```diff
@@ -73,6 +73,14 @@ class ApiUnauthorized(ApiError):
     code = 401
 
 
+class ServiceUnavailable(ApiError):
```

Comment:
@ceorourke We already have `ApiHostError` and `ApiTimeoutError`, how come we need these new ones?

Reply:
lol I ... somehow didn't see those. will update